goss-testing/scripts/python/check_for_unused_drives.py

```diff
@@ -100,11 +101,12 @@ def parse_args():
     logger.debug("Parsing command line arguments: {}".format(sys.argv))
     args = parser.parse_args()
 
-    hw_type = hw_type_map[args.hw_type]    
+    hw_type = hw_type_map[args.hw_type]
+    n_storage_nodes = args.num_storage_nodes
     min_expected_osds = min_osds_per_storage_node[hw_type] * args.num_storage_nodes
     max_expected_osds = max_osds_per_storage_node[hw_type] * args.num_storage_nodes
     logger.info("Based on hardware type ({}) and number of storage nodes ({}): min_expected_osds = {}, max_expected_osds = {}".format(hw_type, args.num_storage_nodes, min_expected_osds, max_expected_osds))
-    return min_expected_osds, max_expected_osds
+    return min_expected_osds, max_expected_osds, n_storage_nodes
 
 def get_num_osds():
     logger.debug("Loading Ceph")
```

Comment:
I think this section needs a rewrite.  one issue is we aren't disconnecting from the cluster.  we need to make sure we close our cluster connection.  we can probably use a better command for the osd num.

Reply:
added a disconnect from cluster. Also return args.num_storage_nodes. did not change anything else as I wanted to make as few changes as possible at this time